src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -56,6 +67,8 @@
 _METHOD_ENUM_TO_STR = {v: k for k, v in _METHOD_STR_TO_ENUM.items()}
 
 PerMethodMetadataType = Mapping[str, Sequence[Tuple[str, str]]]
+# FutureFromCall is both a grpc.Call and grpc.Future
+FutureFromCallType = Any
```

Comment:
> FutureFromCall is both a grpc.Call and grpc.Future

Does it mean `FutureFromCallType` should be a union of these two types, `Union[grpc.Call, grpc.Future]`? Or do you mean it extends both classes?

Reply:
Maybe `_UnaryOutcome` or `_MultiThreadedRendezvous` then?

Reply:
Ahh, apparently there's a guidance on class intersection: https://peps.python.org/pep-0544/#unions-and-intersections-of-protocols:
```suggestion
# FutureFromCall is both a grpc.Call and grpc.Future
class FutureFromCallType(grpc.Call, grpc.Future):
    pass
```

Reply:
Interesting, will create a class then.